dependencies/CMakeLists.txt

```diff
@@ -157,7 +157,6 @@ AddDependency(NAME       simbody
               TAG        Simbody-3.7
               CMAKE_ARGS -DBUILD_EXAMPLES:BOOL=OFF 
```

Comment:
(I can't select the ezc3d line)
Should we bump the ezc3d TAG to your PR in ezc3d [I don't know how you reference a PR in another repo!] already? Maybe it makes more sense to wait, and I can bump the version to the last ezc3d commit

Reply:
Let's wait until you merge the ezc3d PR.

Reply:
I already tested locally that testC3DFileAdapter passes when using my updated ezc3d.